ts/screens/wallet/TransactionsScreen.tsx

```diff
@@ -57,8 +63,9 @@ type OwnProps = NavigationInjectedProps<NavigationParams>;
 
 type Props = OwnProps &
   ReturnType<typeof mapStateToProps> &
-  ReturnType<typeof mapDispatchToProps>;
-
+  ReturnType<typeof mapDispatchToProps> &
+  Readonly<{ isLoading: boolean; loadingCaption?: string | undefined; loadingOpacity?: number | undefined; onCancel?: any }>;
```

Comment:
```suggestion
  Readonly<{ isLoading: boolean }>;
```
I guess this should be enough
btw I can't find any points where this props is used.
If it is true, is `withLoadingSpinner` needed?